src/e2e/utils/helpers.js

```diff
@@ -22,6 +22,10 @@ export const setEnvVariables = async (email) => {
   process.env.E2E_TEST_BASE_URL = ENV_URLS[process.env.E2E_TEST_ENV] || 'https://stage.firefoxmonitor.nonprod.cloudops.mozgcp.net'
 }
 
+export const getBaseUrl = async () => {
+  return ENV_URLS[process.env.E2E_TEST_ENV] || ENV_URLS.local
```

Comment:
Probably won't be an issue in this case, but I think it's a good habit to get into to use `??` instead of `||` for non-boolean values, to avoid accidentally matching on falsy non-boolean values.

```suggestion
  return ENV_URLS[process.env.E2E_TEST_ENV] ?? ENV_URLS.local
```